Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php

```diff
@@ -261,6 +262,13 @@ public function theNodeShouldResolveToUrl(string $nodeAggregateId, string $conte
      */
     public function theNodeShouldNotResolve(string $nodeAggregateId, string $contentStreamId, string $dimensionSpacePoint): void
     {
+        if (
+            ($this->getObject(ConfigurationManager::class)
+                ->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow.mvc.routes')['Neos.Flow'] ?? null) === true
+        ) {
```

Comment:
That check is not exact enough I'm afraid, `Neos.Flow.mvc.routes` could be an array

Reply:
? we access `'Neos.Flow.mvc.routes')['Neos.Flow']`

Reply:
But it _can_ be an array, for example:
```yaml
Neos:
  Flow:
    mvc:
      routes:
        'Neos.Flow':
          position: 'end'
```
Not highly likely, but definitely allowed. Why not check for !null?

Reply:
yeah oke ... or rather !empty as it must not be `false` either